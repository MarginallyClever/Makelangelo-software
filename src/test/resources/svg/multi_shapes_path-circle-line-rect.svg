<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 24.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 595.3 841.9" style="enable-background:new 0 0 595.3 841.9;" xml:space="preserve">
<style type="text/css">
	.st0{fill:none;stroke:#E62227;stroke-miterlimit:10;}
</style>
<path class="st0" d="M68.6,233"/>
<path class="st0" d="M137.1,99.4C94.7,129.5,68.3,181,68.6,233"/>
<path class="st0" d="M171.3,85.6c-12.5,0.4-24.1,6.6-34.2,13.8"/>
<path class="st0" d="M469,544.8"/>
<path class="st0" d="M508.6,480.3c-6.7,24.8-19.6,48.5-39.6,64.5"/>
<path class="st0" d="M515.6,403.9c0.6,25.7-0.3,51.6-7,76.4"/>
<path class="st0" d="M508.8,321.5c4,27.3,6.2,54.8,6.9,82.3"/>
<path class="st0" d="M476.9,257.1c21.4,12.5,28.3,39.9,31.9,64.4"/>
<path class="st0" d="M204.3,104.5c87.1,19.6,168.5,64.2,231.8,127.3"/>
<path class="st0" d="M427.6,553.6"/>
<path class="st0" d="M130.7,375.7c77.5,88.6,181,158,296.9,177.9"/>
<path class="st0" d="M78,257.7c-3.7,44.2,23.6,84.7,52.7,118"/>
<path class="st0" d="M181.8,87.2c5.7,2.4,10.7,6.5,14.4,11.5"/>
<path class="st0" d="M468.9,247.6c-9.2-1.7-18-5.8-25.1-11.9"/>
<path class="st0" d="M457.1,548.5c-7.9-1.2-16-0.6-23.6,1.8"/>
<path class="st0" d="M77.2,248.3c-1.5-2.8-2.9-5.6-4.4-8.4"/>
<circle class="st0" cx="297" cy="420.9" r="164"/>
<line class="st0" x1="418" y1="121.9" x2="255" y2="168.1"/>
<path class="st0" d="M460.6,136.5c-21.1,21.3-42.6,43-69.2,56.9c-26.6,13.8-59.6,18.8-86.6,5.6"/>
<rect x="93" y="603.9" class="st0" width="131" height="131"/>
</svg>
