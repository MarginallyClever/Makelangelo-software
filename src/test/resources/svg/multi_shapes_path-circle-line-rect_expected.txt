[TOOL R0 G0 B0 A255 D1.000, TRAVEL X0.228 Y-0.277, TRAVEL X0.309 Y-0.118, DRAW_LINE X0.309 Y-0.118, DRAW_LINE X0.275 Y-0.149, DRAW_LINE X0.233 Y-0.231, DRAW_LINE X0.228 Y-0.277, TRAVEL X0.350 Y-0.102, DRAW_LINE X0.350 Y-0.102, DRAW_LINE X0.339 Y-0.103, DRAW_LINE X0.319 Y-0.112, DRAW_LINE X0.309 Y-0.118, TRAVEL X0.704 Y-0.647, TRAVEL X0.751 Y-0.570, DRAW_LINE X0.751 Y-0.570, DRAW_LINE X0.743 Y-0.592, DRAW_LINE X0.720 Y-0.631, DRAW_LINE X0.704 Y-0.647, TRAVEL X0.759 Y-0.480, DRAW_LINE X0.759 Y-0.480, DRAW_LINE X0.759 Y-0.503, DRAW_LINE X0.755 Y-0.548, DRAW_LINE X0.751 Y-0.570, TRAVEL X0.751 Y-0.382, DRAW_LINE X0.751 Y-0.382, DRAW_LINE X0.754 Y-0.406, DRAW_LINE X0.758 Y-0.455, DRAW_LINE X0.759 Y-0.480, TRAVEL X0.713 Y-0.305, DRAW_LINE X0.713 Y-0.305, DRAW_LINE X0.729 Y-0.320, DRAW_LINE X0.747 Y-0.360, DRAW_LINE X0.751 Y-0.382, TRAVEL X0.389 Y-0.124, DRAW_LINE X0.389 Y-0.124, DRAW_LINE X0.465 Y-0.147, DRAW_LINE X0.604 Y-0.223, DRAW_LINE X0.664 Y-0.275, TRAVEL X0.654 Y-0.658, TRAVEL X0.302 Y-0.446, DRAW_LINE X0.302 Y-0.446, DRAW_LINE X0.376 Y-0.520, DRAW_LINE X0.554 Y-0.629, DRAW_LINE X0.654 Y-0.658, TRAVEL X0.239 Y-0.306, DRAW_LINE X0.239 Y-0.306, DRAW_LINE X0.242 Y-0.345, DRAW_LINE X0.277 Y-0.415, DRAW_LINE X0.302 Y-0.446, TRAVEL X0.362 Y-0.104, DRAW_LINE X0.362 Y-0.104, DRAW_LINE X0.367 Y-0.106, DRAW_LINE X0.376 Y-0.113, DRAW_LINE X0.379 Y-0.117, TRAVEL X0.703 Y-0.294, DRAW_LINE X0.703 Y-0.294, DRAW_LINE X0.695 Y-0.292, DRAW_LINE X0.680 Y-0.285, DRAW_LINE X0.674 Y-0.280, TRAVEL X0.689 Y-0.652, DRAW_LINE X0.689 Y-0.652, DRAW_LINE X0.682 Y-0.651, DRAW_LINE X0.668 Y-0.652, DRAW_LINE X0.661 Y-0.654, TRAVEL X0.238 Y-0.295, DRAW_LINE X0.238 Y-0.295, DRAW_LINE X0.237 Y-0.292, DRAW_LINE X0.234 Y-0.287, DRAW_LINE X0.233 Y-0.285, TRAVEL X0.694 Y-0.500, DRAW_LINE X0.694 Y-0.503, DRAW_LINE X0.694 Y-0.507, DRAW_LINE X0.694 Y-0.510, DRAW_LINE X0.694 Y-0.514, DRAW_LINE X0.693 Y-0.517, DRAW_LINE X0.693 Y-0.520, DRAW_LINE X0.693 Y-0.524, DRAW_LINE X0.692 Y-0.527, DRAW_LINE X0.692 Y-0.530, DRAW_LINE X0.691 Y-0.534, DRAW_LINE X0.690 Y-0.537, DRAW_LINE X0.690 Y-0.540, DRAW_LINE X0.689 Y-0.544, DRAW_LINE X0.688 Y-0.547, DRAW_LINE X0.687 Y-0.550, DRAW_LINE X0.686 Y-0.554, DRAW_LINE X0.686 Y-0.557, DRAW_LINE X0.684 Y-0.560, DRAW_LINE X0.683 Y-0.563, DRAW_LINE X0.682 Y-0.567, DRAW_LINE X0.681 Y-0.570, DRAW_LINE X0.680 Y-0.573, DRAW_LINE X0.679 Y-0.576, DRAW_LINE X0.677 Y-0.579, DRAW_LINE X0.676 Y-0.582, DRAW_LINE X0.674 Y-0.585, DRAW_LINE X0.673 Y-0.588, DRAW_LINE X0.671 Y-0.591, DRAW_LINE X0.670 Y-0.594, DRAW_LINE X0.668 Y-0.597, DRAW_LINE X0.666 Y-0.600, DRAW_LINE X0.664 Y-0.603, DRAW_LINE X0.663 Y-0.606, DRAW_LINE X0.661 Y-0.609, DRAW_LINE X0.659 Y-0.612, DRAW_LINE X0.657 Y-0.614, DRAW_LINE X0.655 Y-0.617, DRAW_LINE X0.653 Y-0.620, DRAW_LINE X0.651 Y-0.623, DRAW_LINE X0.648 Y-0.625, DRAW_LINE X0.646 Y-0.628, DRAW_LINE X0.644 Y-0.630, DRAW_LINE X0.642 Y-0.633, DRAW_LINE X0.639 Y-0.635, DRAW_LINE X0.637 Y-0.638, DRAW_LINE X0.635 Y-0.640, DRAW_LINE X0.632 Y-0.642, DRAW_LINE X0.630 Y-0.645, DRAW_LINE X0.627 Y-0.647, DRAW_LINE X0.624 Y-0.649, DRAW_LINE X0.622 Y-0.651, DRAW_LINE X0.619 Y-0.653, DRAW_LINE X0.616 Y-0.656, DRAW_LINE X0.614 Y-0.658, DRAW_LINE X0.611 Y-0.660, DRAW_LINE X0.608 Y-0.661, DRAW_LINE X0.605 Y-0.663, DRAW_LINE X0.602 Y-0.665, DRAW_LINE X0.600 Y-0.667, DRAW_LINE X0.597 Y-0.669, DRAW_LINE X0.594 Y-0.670, DRAW_LINE X0.591 Y-0.672, DRAW_LINE X0.588 Y-0.674, DRAW_LINE X0.585 Y-0.675, DRAW_LINE X0.582 Y-0.676, DRAW_LINE X0.578 Y-0.678, DRAW_LINE X0.575 Y-0.679, DRAW_LINE X0.572 Y-0.681, DRAW_LINE X0.569 Y-0.682, DRAW_LINE X0.566 Y-0.683, DRAW_LINE X0.563 Y-0.684, DRAW_LINE X0.559 Y-0.685, DRAW_LINE X0.556 Y-0.686, DRAW_LINE X0.553 Y-0.687, DRAW_LINE X0.550 Y-0.688, DRAW_LINE X0.546 Y-0.689, DRAW_LINE X0.543 Y-0.690, DRAW_LINE X0.540 Y-0.690, DRAW_LINE X0.536 Y-0.691, DRAW_LINE X0.533 Y-0.692, DRAW_LINE X0.530 Y-0.692, DRAW_LINE X0.526 Y-0.693, DRAW_LINE X0.523 Y-0.693, DRAW_LINE X0.520 Y-0.694, DRAW_LINE X0.516 Y-0.694, DRAW_LINE X0.513 Y-0.694, DRAW_LINE X0.509 Y-0.694, DRAW_LINE X0.506 Y-0.695, DRAW_LINE X0.503 Y-0.695, DRAW_LINE X0.499 Y-0.695, DRAW_LINE X0.496 Y-0.695, DRAW_LINE X0.492 Y-0.695, DRAW_LINE X0.489 Y-0.694, DRAW_LINE X0.486 Y-0.694, DRAW_LINE X0.482 Y-0.694, DRAW_LINE X0.479 Y-0.694, DRAW_LINE X0.475 Y-0.693, DRAW_LINE X0.472 Y-0.693, DRAW_LINE X0.469 Y-0.692, DRAW_LINE X0.465 Y-0.692, DRAW_LINE X0.462 Y-0.691, DRAW_LINE X0.459 Y-0.690, DRAW_LINE X0.455 Y-0.690, DRAW_LINE X0.452 Y-0.689, DRAW_LINE X0.449 Y-0.688, DRAW_LINE X0.446 Y-0.687, DRAW_LINE X0.442 Y-0.686, DRAW_LINE X0.439 Y-0.685, DRAW_LINE X0.436 Y-0.684, DRAW_LINE X0.433 Y-0.683, DRAW_LINE X0.429 Y-0.682, DRAW_LINE X0.426 Y-0.681, DRAW_LINE X0.423 Y-0.679, DRAW_LINE X0.420 Y-0.678, DRAW_LINE X0.417 Y-0.676, DRAW_LINE X0.414 Y-0.675, DRAW_LINE X0.411 Y-0.674, DRAW_LINE X0.408 Y-0.672, DRAW_LINE X0.405 Y-0.670, DRAW_LINE X0.402 Y-0.669, DRAW_LINE X0.399 Y-0.667, DRAW_LINE X0.396 Y-0.665, DRAW_LINE X0.393 Y-0.663, DRAW_LINE X0.390 Y-0.661, DRAW_LINE X0.387 Y-0.660, DRAW_LINE X0.385 Y-0.658, DRAW_LINE X0.382 Y-0.656, DRAW_LINE X0.379 Y-0.653, DRAW_LINE X0.377 Y-0.651, DRAW_LINE X0.374 Y-0.649, DRAW_LINE X0.371 Y-0.647, DRAW_LINE X0.369 Y-0.645, DRAW_LINE X0.366 Y-0.642, DRAW_LINE X0.364 Y-0.640, DRAW_LINE X0.361 Y-0.638, DRAW_LINE X0.359 Y-0.635, DRAW_LINE X0.357 Y-0.633, DRAW_LINE X0.354 Y-0.630, DRAW_LINE X0.352 Y-0.628, DRAW_LINE X0.350 Y-0.625, DRAW_LINE X0.348 Y-0.623, DRAW_LINE X0.346 Y-0.620, DRAW_LINE X0.344 Y-0.617, DRAW_LINE X0.342 Y-0.614, DRAW_LINE X0.340 Y-0.612, DRAW_LINE X0.338 Y-0.609, DRAW_LINE X0.336 Y-0.606, DRAW_LINE X0.334 Y-0.603, DRAW_LINE X0.332 Y-0.600, DRAW_LINE X0.331 Y-0.597, DRAW_LINE X0.329 Y-0.594, DRAW_LINE X0.327 Y-0.591, DRAW_LINE X0.326 Y-0.588, DRAW_LINE X0.324 Y-0.585, DRAW_LINE X0.323 Y-0.582, DRAW_LINE X0.321 Y-0.579, DRAW_LINE X0.320 Y-0.576, DRAW_LINE X0.319 Y-0.573, DRAW_LINE X0.317 Y-0.570, DRAW_LINE X0.316 Y-0.567, DRAW_LINE X0.315 Y-0.563, DRAW_LINE X0.314 Y-0.560, DRAW_LINE X0.313 Y-0.557, DRAW_LINE X0.312 Y-0.554, DRAW_LINE X0.311 Y-0.550, DRAW_LINE X0.310 Y-0.547, DRAW_LINE X0.309 Y-0.544, DRAW_LINE X0.309 Y-0.540, DRAW_LINE X0.308 Y-0.537, DRAW_LINE X0.307 Y-0.534, DRAW_LINE X0.307 Y-0.530, DRAW_LINE X0.306 Y-0.527, DRAW_LINE X0.306 Y-0.524, DRAW_LINE X0.305 Y-0.520, DRAW_LINE X0.305 Y-0.517, DRAW_LINE X0.305 Y-0.514, DRAW_LINE X0.305 Y-0.510, DRAW_LINE X0.305 Y-0.507, DRAW_LINE X0.304 Y-0.503, DRAW_LINE X0.304 Y-0.500, DRAW_LINE X0.304 Y-0.497, DRAW_LINE X0.305 Y-0.493, DRAW_LINE X0.305 Y-0.490, DRAW_LINE X0.305 Y-0.486, DRAW_LINE X0.305 Y-0.483, DRAW_LINE X0.305 Y-0.480, DRAW_LINE X0.306 Y-0.476, DRAW_LINE X0.306 Y-0.473, DRAW_LINE X0.307 Y-0.469, DRAW_LINE X0.307 Y-0.466, DRAW_LINE X0.308 Y-0.463, DRAW_LINE X0.309 Y-0.459, DRAW_LINE X0.309 Y-0.456, DRAW_LINE X0.310 Y-0.453, DRAW_LINE X0.311 Y-0.450, DRAW_LINE X0.312 Y-0.446, DRAW_LINE X0.313 Y-0.443, DRAW_LINE X0.314 Y-0.440, DRAW_LINE X0.315 Y-0.437, DRAW_LINE X0.316 Y-0.433, DRAW_LINE X0.317 Y-0.430, DRAW_LINE X0.319 Y-0.427, DRAW_LINE X0.320 Y-0.424, DRAW_LINE X0.321 Y-0.421, DRAW_LINE X0.323 Y-0.418, DRAW_LINE X0.324 Y-0.415, DRAW_LINE X0.326 Y-0.412, DRAW_LINE X0.327 Y-0.408, DRAW_LINE X0.329 Y-0.406, DRAW_LINE X0.331 Y-0.403, DRAW_LINE X0.332 Y-0.400, DRAW_LINE X0.334 Y-0.397, DRAW_LINE X0.336 Y-0.394, DRAW_LINE X0.338 Y-0.391, DRAW_LINE X0.340 Y-0.388, DRAW_LINE X0.342 Y-0.385, DRAW_LINE X0.344 Y-0.383, DRAW_LINE X0.346 Y-0.380, DRAW_LINE X0.348 Y-0.377, DRAW_LINE X0.350 Y-0.375, DRAW_LINE X0.352 Y-0.372, DRAW_LINE X0.354 Y-0.370, DRAW_LINE X0.357 Y-0.367, DRAW_LINE X0.359 Y-0.365, DRAW_LINE X0.361 Y-0.362, DRAW_LINE X0.364 Y-0.360, DRAW_LINE X0.366 Y-0.357, DRAW_LINE X0.369 Y-0.355, DRAW_LINE X0.371 Y-0.353, DRAW_LINE X0.374 Y-0.351, DRAW_LINE X0.377 Y-0.349, DRAW_LINE X0.379 Y-0.346, DRAW_LINE X0.382 Y-0.344, DRAW_LINE X0.385 Y-0.342, DRAW_LINE X0.387 Y-0.340, DRAW_LINE X0.390 Y-0.338, DRAW_LINE X0.393 Y-0.337, DRAW_LINE X0.396 Y-0.335, DRAW_LINE X0.399 Y-0.333, DRAW_LINE X0.402 Y-0.331, DRAW_LINE X0.405 Y-0.330, DRAW_LINE X0.408 Y-0.328, DRAW_LINE X0.411 Y-0.326, DRAW_LINE X0.414 Y-0.325, DRAW_LINE X0.417 Y-0.323, DRAW_LINE X0.420 Y-0.322, DRAW_LINE X0.423 Y-0.321, DRAW_LINE X0.426 Y-0.319, DRAW_LINE X0.429 Y-0.318, DRAW_LINE X0.433 Y-0.317, DRAW_LINE X0.436 Y-0.316, DRAW_LINE X0.439 Y-0.315, DRAW_LINE X0.442 Y-0.314, DRAW_LINE X0.446 Y-0.313, DRAW_LINE X0.449 Y-0.312, DRAW_LINE X0.452 Y-0.311, DRAW_LINE X0.455 Y-0.310, DRAW_LINE X0.459 Y-0.309, DRAW_LINE X0.462 Y-0.309, DRAW_LINE X0.465 Y-0.308, DRAW_LINE X0.469 Y-0.308, DRAW_LINE X0.472 Y-0.307, DRAW_LINE X0.475 Y-0.307, DRAW_LINE X0.479 Y-0.306, DRAW_LINE X0.482 Y-0.306, DRAW_LINE X0.486 Y-0.306, DRAW_LINE X0.489 Y-0.305, DRAW_LINE X0.492 Y-0.305, DRAW_LINE X0.496 Y-0.305, DRAW_LINE X0.499 Y-0.305, DRAW_LINE X0.503 Y-0.305, DRAW_LINE X0.506 Y-0.305, DRAW_LINE X0.509 Y-0.305, DRAW_LINE X0.513 Y-0.306, DRAW_LINE X0.516 Y-0.306, DRAW_LINE X0.520 Y-0.306, DRAW_LINE X0.523 Y-0.307, DRAW_LINE X0.526 Y-0.307, DRAW_LINE X0.530 Y-0.308, DRAW_LINE X0.533 Y-0.308, DRAW_LINE X0.536 Y-0.309, DRAW_LINE X0.540 Y-0.309, DRAW_LINE X0.543 Y-0.310, DRAW_LINE X0.546 Y-0.311, DRAW_LINE X0.550 Y-0.312, DRAW_LINE X0.553 Y-0.313, DRAW_LINE X0.556 Y-0.314, DRAW_LINE X0.559 Y-0.315, DRAW_LINE X0.563 Y-0.316, DRAW_LINE X0.566 Y-0.317, DRAW_LINE X0.569 Y-0.318, DRAW_LINE X0.572 Y-0.319, DRAW_LINE X0.575 Y-0.321, DRAW_LINE X0.578 Y-0.322, DRAW_LINE X0.582 Y-0.323, DRAW_LINE X0.585 Y-0.325, DRAW_LINE X0.588 Y-0.326, DRAW_LINE X0.591 Y-0.328, DRAW_LINE X0.594 Y-0.330, DRAW_LINE X0.597 Y-0.331, DRAW_LINE X0.600 Y-0.333, DRAW_LINE X0.602 Y-0.335, DRAW_LINE X0.605 Y-0.337, DRAW_LINE X0.608 Y-0.338, DRAW_LINE X0.611 Y-0.340, DRAW_LINE X0.614 Y-0.342, DRAW_LINE X0.616 Y-0.344, DRAW_LINE X0.619 Y-0.346, DRAW_LINE X0.622 Y-0.349, DRAW_LINE X0.624 Y-0.351, DRAW_LINE X0.627 Y-0.353, DRAW_LINE X0.630 Y-0.355, DRAW_LINE X0.632 Y-0.357, DRAW_LINE X0.635 Y-0.360, DRAW_LINE X0.637 Y-0.362, DRAW_LINE X0.639 Y-0.365, DRAW_LINE X0.642 Y-0.367, DRAW_LINE X0.644 Y-0.370, DRAW_LINE X0.646 Y-0.372, DRAW_LINE X0.648 Y-0.375, DRAW_LINE X0.651 Y-0.377, DRAW_LINE X0.653 Y-0.380, DRAW_LINE X0.655 Y-0.383, DRAW_LINE X0.657 Y-0.385, DRAW_LINE X0.659 Y-0.388, DRAW_LINE X0.661 Y-0.391, DRAW_LINE X0.663 Y-0.394, DRAW_LINE X0.664 Y-0.397, DRAW_LINE X0.666 Y-0.400, DRAW_LINE X0.668 Y-0.403, DRAW_LINE X0.670 Y-0.406, DRAW_LINE X0.671 Y-0.408, DRAW_LINE X0.673 Y-0.412, DRAW_LINE X0.674 Y-0.415, DRAW_LINE X0.676 Y-0.418, DRAW_LINE X0.677 Y-0.421, DRAW_LINE X0.679 Y-0.424, DRAW_LINE X0.680 Y-0.427, DRAW_LINE X0.681 Y-0.430, DRAW_LINE X0.682 Y-0.433, DRAW_LINE X0.683 Y-0.437, DRAW_LINE X0.684 Y-0.440, DRAW_LINE X0.686 Y-0.443, DRAW_LINE X0.686 Y-0.446, DRAW_LINE X0.687 Y-0.450, DRAW_LINE X0.688 Y-0.453, DRAW_LINE X0.689 Y-0.456, DRAW_LINE X0.690 Y-0.459, DRAW_LINE X0.690 Y-0.463, DRAW_LINE X0.691 Y-0.466, DRAW_LINE X0.692 Y-0.469, DRAW_LINE X0.692 Y-0.473, DRAW_LINE X0.693 Y-0.476, DRAW_LINE X0.693 Y-0.480, DRAW_LINE X0.693 Y-0.483, DRAW_LINE X0.694 Y-0.486, DRAW_LINE X0.694 Y-0.490, DRAW_LINE X0.694 Y-0.493, DRAW_LINE X0.694 Y-0.497, DRAW_LINE X0.694 Y-0.500, TRAVEL X0.643 Y-0.145, DRAW_LINE X0.449 Y-0.200, TRAVEL X0.694 Y-0.162, DRAW_LINE X0.694 Y-0.162, DRAW_LINE X0.675 Y-0.181, DRAW_LINE X0.634 Y-0.216, DRAW_LINE X0.611 Y-0.230, DRAW_LINE X0.611 Y-0.230, DRAW_LINE X0.586 Y-0.240, DRAW_LINE X0.534 Y-0.244, DRAW_LINE X0.508 Y-0.236, TRAVEL X0.257 Y-0.717, DRAW_LINE X0.413 Y-0.717, DRAW_LINE X0.413 Y-0.873, DRAW_LINE X0.257 Y-0.873, DRAW_LINE X0.257 Y-0.717]